config/config.yaml

```diff
@@ -11,114 +11,170 @@
 #         link: jquery-ui.min.css filename or null
 #         autoload: true (defaults to false)
 assets:
```

Comment:
alphabetical makes sense, but aren't some of these dependent on order.
For example, doesn't jquery need to ones that depend on jquery?

Reply:
hrm. good point. I'll have a look at Header.php to see how these things are being loaded. I'd prefer to keep this config in order and use the Header file to manage what loads in what order. Most script managers do this by having a "depends:" key that only loads a script after all of its dependencies have been loaded first.